dingo/pulp_based_impl.py

```diff
@@ -0,0 +1,449 @@
+# dingo : a python library for metabolic networks sampling and analysis
+# dingo is part of GeomScale project
+
+# Copyright (c) 2024 Ke Shi
+
+# Licensed under GNU LGPL.3, see LICENCE file
+
+import pulp as pl
+import numpy as np
+import math
+import sys
+
+solver_name = "HiGHS"
```

Comment:
I think the solver_name should be a function parameter in `fba(), fva()` etc with default value = "HiGHS"
Then in the unit tests we should call all those functions with default parameter and with "gurobi" (and maybe test more in the future?).